tools/jira_client/jira_dryrun_client.py

```diff
@@ -0,0 +1,31 @@
+from .consts import CLOSED_STATUS
+
+
+class JiraDryRunClient:
+    def __init__(self, jira_client, dry_run_stream) -> None:
+        self._dry_run_stream = dry_run_stream
+        self._jira_client = jira_client
+
+    def __getattr__(self, name):
+        try:
+            return object.__getattribute__(self, name)
+        except AttributeError:
+            return getattr(self._jira_client, name)
+
+    def create_issue_link(self, link_name, issue, root_issue):
+        print(
+            f"Linked issue key={issue.key} as {link_name} root issue key={root_issue.key}\n",
+            file=self._dry_run_stream,
+            flush=True,
+        )
+
+    def transition_issue(self, issue, target_transition_id):
+        transition_names = {CLOSED_STATUS: "Closed"}
+        transition_name = "Unknown"
+        if target_transition_id in transition_names.keys():
+            transition_name = transition_names[target_transition_id]
+        print(
+            f"Transitioned issue key={issue.issue.key} to {transition_name}({target_transition_id})",
+            file=self._dry_run_stream,
+            flush=True,
+        )
```

Comment:
why masking the status? I think it can just be:
```suggestion
        print(
            f"Transitioned issue key={issue.issue.key} to {target_transition_id},
            file=self._dry_run_stream,
            flush=True,
        )
```

Reply:
Yeah, you are right, leftover from status_id and blindly replaced :sweat: